pages/1.7/administration/release-notes/1-7/index.md

```diff
@@ -13,13 +13,13 @@ The release notes provide a list of useful topics and links for DC/OS.
 
 You can now install packages from the DC/OS Universe with a single click in the web interface. The packages can be installed with defaults or customized directly in the UI. For more information, see the [documentation][1].
 
-![UI Universe](/assets/images/ui-universe-ee.gif)
+![UI Universe](/administration/img/ui-universe-ee.gif)
```

Comment:
These are still broken in my local build. Perhaps the path doesn't originate at the correct place?